internal/brokers/broker.go

```diff
@@ -42,6 +42,8 @@ type brokerer interface {
 	IsAuthenticated(ctx context.Context, sessionID, authenticationData string) (access, data string, err error)
 	EndSession(ctx context.Context, sessionID string) (err error)
 	CancelIsAuthenticated(ctx context.Context, sessionID string)
+
```

Comment:
I thought about it as well. The issue with putting it first is that it would look like it's "expected" behaviour, but it's not something that should happen on every authentication flow but a few. Does that make sense?